collector/collector_manager_test.go

```diff
@@ -14,6 +14,7 @@
 
 package collector
 
+/*
```

Comment:
There was a name collision between the Collector struct and the Collector interface(temporarily renamed as CollectorInterface). Once we finalize on the names for these, we can make the necessary changes in all places(eg manager.go uses Collector as an interface) and put back the test.